test_unstructured/partition/docx/test_doc.py

```diff
@@ -44,6 +46,15 @@ def test_partition_doc_from_filename(mock_document, expected_elements, tmpdir, c
     assert capsys.readouterr().err == ""
 
 
+@pytest.mark.skipif(is_in_docker, reason="Skipping this test in Docker container")
```

Comment:
This was not finding the tmpfile in the docker container for some reason. Since we're testing the `partition_doc` else where in this suite, wasn't overly concerned but would like to figure out why it's happening.